boards/starfive/visionfive2/doc/index.rst

```diff
@@ -0,0 +1,74 @@
+.. _visionfive2:
+
+Starfive VisionFive2 JH7110
+###########################
+
+Overview
+********
+
+Starfive VisionFive 2 is the world’s first high-performance RISC-V single board computer (SBC) with an integrated GPU. Compared with its last generation, VisionFive 2 has been fully upgraded with significant improvements in the processor work frequency, multimedia processing capabilities, scalability, etc. Its superior performance and reasonable price make VisionFive 2 the best affordable RISC-V development board ever.
+
+.. figure:: img/Visionfive2.webp
+   :align: center
+   :alt:
+
+Programming and debugging
+*************************
+
+Building
+========
+
+Applications for the ``visionfive2_jh7110`` board configuration can be built
+as usual (see :ref:`build_an_application`) using the corresponding board name:
+
+.. zephyr-app-commands::
+   :board: visionfive2
+   :goals: build
+
+To build zephyr.bin for this board use this command:
+
+.. code-block:: console
+
+   west build -p always -b visionfive2 samples/hello_world/
+
+spl_tool is a jh7110 signature tool used to generate spl header information and generate zephyr.bin.normal.out.
+`here <https://github.com/starfive-tech/Tools/tree/master/spl_tool/>`_
+
+.. code-block:: console
+
+   ./spl_tool -c -f build/zephyr/zephyr.bin
```

Comment:
This seems to imply one would download the tool at the root of their Zephyr source tree?

Reply:
Doesn't matter if it's in root of zephyr source tree or not. The tool can be cloned anywhere, all that it is needed for is to create spl header for zephyr binary.